ui/page/discover/internal/livestreamSection/view.jsx

```diff
@@ -95,7 +95,7 @@ export default function LivestreamSection(props: Props) {
         />
 
         {liveTilesOverLimit && liveSection === SECTION.COLLAPSED && (
-          <div className="livestream-list--view-more">
+          <div className="upcoming-list__view-more">
```

Comment:
I know and I agree. I didn't change it already because the parent is actually a grid and not a list. Definitely not ideal, but using upcoming instead of livestream is at least less confusing for now.